### PR TITLE
Add new advertising guidelines

### PR DESCRIPTION
**Pull request by joseluisbolos**

### CODE REVIEW

File: practices/advertising.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# Advertising
+
+These guidelines are based on ethical, [accessibility](accessibility.md) or performance concerns.
```
Comment (hollsk) [inline practices/advertising.md]: Replacing "concerns" with "recommendations" will be more in line with your aim to keep the wording positive here

File: practices/advertising.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+* Don't advertise products with obvious exaggerations or dubious claims that are impossible to prove.
+* Don't advertise "free" products or services that have hidden costs or charges.
+* Don't advertise gambling, alcohol and tobacco.
+* Don't use [pop-under ads](https://en.wikipedia.org/wiki/Pop-up_ad#Pop-under_ads).
+* Adverts must not contain spyware, malware or any software that results in an unexpected or deceptive experience. This includes links to sites containing these products.
+
+
+## Accessibility
+
+* Don't use ads that make heavy use of fast animations or flashing content.
```
Comment (hollsk) [inline practices/advertising.md]: Maybe remove the qualifier "heavy" from this sentence so as not to open the door for a discussion around what constitutes heavy use

File: practices/advertising.md (hunk 1 above)
Comment (joseluisbolos) [inline practices/advertising.md]: Done

File: practices/advertising.md (hunk 2 above)
Comment (joseluisbolos) [inline practices/advertising.md]: ... and done!

**Pull request merged**